gravityflow/gravityflow
# Replace old spinner gif with CSS one

## Description
The Status page references the Gravity Forms spinner.gif, which was removed in GF 2.5. This both caused a 404 on the page, and broke our loading state. 

This fix removes references to the .gif, and instead uses a CSS-only loading state. 

Addresses issue 63: https://github.com/gravityflow/backlog/issues/63

## Testing instructions
1. Go to the GravityFlow status page
2. Select some items, then hit "Export". 
3. Ensure you see a well-formatted loading spinner. (Note: to help testing, use the Dev Console's tool to throttle your speed down to Slow 3G so you can see the spinner for longer).
 
## Screenshots
![Screenshot on 2021-02-07 at 14-47-32](https://user-images.githubusercontent.com/6209061/107159058-7a4cfd00-6953-11eb-882f-7a5c409854c8.png)


## Checklist:
